vowpalwabbit/nn.cc

```diff
@@ -225,7 +223,7 @@ void predict_or_learn_multi(nn& n, single_learner& base, example& ec)
 
   CONVERSE:  // That's right, I'm using goto.  So sue me.
 
-    n.output_layer.total_sum_feat_sq = 1;
+    n.output_layer.reset_total_sum_feat_sq();
```

Comment:
The original code sets the sum_feat_sq to 1 while the function sets it to 0. Is that ok?

Reply:
Yes, since it will get calculated from scratch when needed and that 0 wont make it out